app/controllers/policies_controller.rb

```diff
@@ -1,12 +1,17 @@
 class PoliciesController < ApplicationController
+  before_action :authenticate_user!
   before_action :policies_params, only: [:create, :update]
   before_action :set_policy, only: [:edit, :update , :destroy]
   def new
-    @policies = Policy.new
+    @policy = Policy.new
   end
 
   def index
-    @policies = Policy.all
+    if (params[:search])
+      @policies = Policy.search(params[:search]).paginate(per_page: 2, page: params[:page]).order(created_at: :asc)
+    else
+      @policies = Policy.all.paginate(per_page: 2, page: params[:page]).order(created_at: :asc)
```

Comment:
@KaranBamaniya-BTC :.paginate(per_page: 2, page: params[:page]).order(created_at: :asc) is duplicate, 
take the help of @RaviMariya-BTC  for pagination